check_slack_connectivity.py

```diff
@@ -10,7 +8,5 @@
 # 2) an announcement to the Slack channel that there's a new board running
 #     canary tests.
 if __name__ == "__main__":
-    client = slack_sdk.WebClient(token=config.auth_token)
-    text = (f"Canary tests on the {config.board_name} board should be able " +
-            "to report to Slack!")
-    result = client.chat_postMessage(channel=config.channel_id, text=text)
+    text = "the canary tests should be able to report to Slack!"
```

Comment:
I notice you removed the board name from this message, will you be able to see what board it is from something else?

Reply:
nvm I see the board name is in the preamble now

Reply:
Yeah, I didn't like the duplication, though I'm not sure I'll like the new message format, either. If you've got suggestions of what to do instead, I'm interested!